src/Umbraco.Web/PublishedCache/NuCache/PublishedSnapshotService.cs

```diff
@@ -128,10 +128,12 @@ internal class PublishedSnapshotService : PublishedSnapshotServiceBase
                         var path = GetLocalFilesPath();
                         var localContentDbPath = Path.Combine(path, "NuCache.Content.db");
                         var localMediaDbPath = Path.Combine(path, "NuCache.Media.db");
-                        _localDbExists = File.Exists(localContentDbPath) && File.Exists(localMediaDbPath);
```

Comment:
You've removed using the flag `_localDbExists` but the actual variable still exists

Reply:
_localDbExists is still used elsewhere by LoadCachesOnStartup , so keeping this value the same as present to not affect behaviour elsewhere.

Reply:
However, you've updated how the 2 different stores are checked and then we only set _localDbExists to true if **both** exist, later we check this flag in `LoadCaches`. It would seem to me that `LoadCaches` should use individual checks of whether the media or content caches exist and proceed accordingly instead of using this single field. I think it would make more sense to have the fields: `_localMediaDbExists` and `_localContentDbExists` and use these fields in place of wherever `_localDbExists` is used and remove `_localDbExists` entirely.

Reply:
agreed